integration-tests/sample_test.go

```diff
@@ -58,3 +63,147 @@ func TestAuth(t *testing.T) {
 		t.Fatal("UserID  mismatch")
 	}
 }
+
+func TestValidateSignature(t *testing.T) {
+	if options.proxyHost == "" || options.proxyPort == "" || options.sampleName == "" {
+		t.Skip("sample-name, proxy-host or proxy-port are missing => skip TestValidateSignature")
+	}
```

Comment:
Let's make this explicit.

Reply:
Indeed, I thought it's better to skip, not fail, as we have two jobs _run-integration-test_ and _run-integration-tests-with-proxy_ and it make sense only the second one to run the new test.